src/pe/import.rs

```diff
@@ -225,7 +224,8 @@ impl<'a> SyntheticImportDirectoryEntry<'a> {
     ) -> error::Result<SyntheticImportDirectoryEntry<'a>> {
         const LE: scroll::Endian = scroll::LE;
         let name_rva = import_directory_entry.name_rva;
-        let name = utils::try_name(bytes, name_rva as usize, sections, file_alignment, opts)?;
+        let name = utils::try_name(bytes, name_rva as usize, sections, file_alignment, opts)
```

Comment:
Why does this like get a default now ? Also won’t this be an empty string ? If the name can fail we don’t usually return an empty string, but fail. If the name is optional then we return None usually